osu.Game.Modes.Osu/Objects/Spinner.cs

```diff
@@ -5,5 +5,8 @@ namespace osu.Game.Modes.Osu.Objects
 {
     public class Spinner : OsuHitObject
     {
+        public double EndPoint;

```

Comment:
don't like this variable name. we'll probably want to change the base implementation of EndTime or Duration to allow setting it directly.